src/Core.jl

```diff
@@ -327,6 +295,7 @@ function EvaluatePoint!(p::DSProblem{FT}, trial_points::Vector{Vector{FT}})::Ite
         #Point is feasible for relaxed constraints, so evaluate it
         t1 = time()
         cost = function_evaluation(p, point)
+        evals += 1
```

Comment:
What is this variable for? It is incremented on every trial point, but I can't find where it is consumed.

Reply:
This was previously used for tracking function evaluations, but this functionality has been moved to the objective evaluation function and the status type.

Reply:
Removed in 0d7eb1f.